### ISSUE TITLE
A question about the colors used in atom-one-dark-theme.el

### ISSUE THREAD

**Issue by ghost** [jonathanchu/atom-one-dark-theme#15]
Hi, 

I was browsing through the source code of [one-dark-syntax](https://github.com/atom/one-dark-syntax/) and I couldn't find the following colors that have been used in [atom-one-dark-theme](https://github.com/jonathanchu/atom-one-dark-theme):

https://github.com/jonathanchu/atom-one-dark-theme/blob/master/atom-one-dark-theme.el#L22
https://github.com/jonathanchu/atom-one-dark-theme/blob/master/atom-one-dark-theme.el#L23
https://github.com/jonathanchu/atom-one-dark-theme/blob/master/atom-one-dark-theme.el#L36
https://github.com/jonathanchu/atom-one-dark-theme/blob/master/atom-one-dark-theme.el#L37

Is it the case that I wasn't able to find these colors in the [one-dark-syntax](https://github.com/atom/one-dark-syntax/) repo or did you use these colors independently?

Thanks,
sixpointzero

**Comment by jonathanchu** [comment]
Hey @sixpointzero!  Hmm, it could be one of three things here:

1) The official one-dark-syntax color was modified since porting over this theme. If that's the case, these can be updated on our end.
2) It was my closest approximation to the color match and might be slightly off, hence the color not found.
3) I messed up :)

I can take a look at these in more detail tonight and verify the colors.  However, feel free to correct these if they don't seem right to you!

**Comment by jonathanchu** [comment]
Whoops, accidentally closed this one out.  Re-opened.

**Comment by jakubdonovan** [comment]
I cannot find this package in melpa. Where did it go and was this fixed?

**Comment by jonathanchu** [comment]
@jakubdonovan hey!  I haven't verified if these colors are exact or up-to-date with the latest official Atom One Dark theme yet.

And also, yes, it is still on MELPA - https://melpa.org/#/atom-one-dark-theme

**Comment by jhacksworth** [comment]
**TL;DR**: The only color variable that I can't account for is `atom-one-dark-bg-1`.

I amended the links in @sixpointzero's [original question](https://github.com/jonathanchu/atom-one-dark-theme/issues/15#issue-158142367) to point to the state the repo was in at the time of the question, Jun 2, 2016, instead of *master*, so that we can understand what he's referring to. I'm also including the names of the color variables in the link names, for ease of reference:

(These links are to an outdated version of this theme)
- [`atom-one-dark-bg-1`](https://github.com/jonathanchu/atom-one-dark-theme/blob/807679bfe9ddc53a0cd6cfcdcd851b4496c707e3/atom-one-dark-theme.el#L22)
- [`atom-one-dark-hl`](https://github.com/jonathanchu/atom-one-dark-theme/blob/807679bfe9ddc53a0cd6cfcdcd851b4496c707e3/atom-one-dark-theme.el#L23)
- [`atom-one-dark-silver`](https://github.com/jonathanchu/atom-one-dark-theme/blob/807679bfe9ddc53a0cd6cfcdcd851b4496c707e3/atom-one-dark-theme.el#L36)
- [`atom-one-dark-black`](https://github.com/jonathanchu/atom-one-dark-theme/blob/807679bfe9ddc53a0cd6cfcdcd851b4496c707e3/atom-one-dark-theme.el#L37)

Here is the current version of the theme at the time of this writing, on Jan 31, 2018 (Note that the values for `atom-one-dark-silver` and `atom-one-dark-black` have been updated):
- [`atom-one-dark-bg-1`](https://github.com/jonathanchu/atom-one-dark-theme/blob/fbe026c64f53bf5afa27c55fda6118d45cea7d5e/atom-one-dark-theme.el#L37)
- [`atom-one-dark-hl`](https://github.com/jonathanchu/atom-one-dark-theme/blob/fbe026c64f53bf5afa27c55fda6118d45cea7d5e/atom-one-dark-theme.el#L38)
- [`atom-one-dark-silver`](https://github.com/jonathanchu/atom-one-dark-theme/blob/fbe026c64f53bf5afa27c55fda6118d45cea7d5e/atom-one-dark-theme.el#L52)
- [`atom-one-dark-black`](https://github.com/jonathanchu/atom-one-dark-theme/blob/fbe026c64f53bf5afa27c55fda6118d45cea7d5e/atom-one-dark-theme.el#L53)

I created a gist with a mapping of Atom One Dark colors to this theme's colors in this gist:

- https://gist.github.com/jhacksworth/889f7cf8b09efd81ef0b1b69b3ef8c0a

Here is a sample output of that mapping (when plugged into the LESS CSS compiler):

- https://gist.github.com/jhacksworth/e7efbe1a5cfe53abfe7619a3cdbf0f07

The only color variable that I can't account for is `atom-one-dark-bg-1`. If we want to only use colors defined in the Atom theme, I think that a good candidate for `atom-one-dark-bg-1` is `@input-background-color`, which has a hex color value of: `#1b1d23`

I hope this makes sense and isn't a confusing mess. 😃
